account_chart_update/tests/test_account_chart_update.py

```diff
@@ -466,3 +466,14 @@ def test_00_matching(self):
         self.assertTrue(list(self.fp.get_external_id().values())[0])
         self.assertEqual(fp_rec.get_external_id().get(fp_id), expected_xmlid)
         wizard.unlink()
+
+    def test_01_archived_fiscal_position(self):
+        # Test wizard won't duplicate existing fiscal positions when archived
+        fiscal_position = self.env["account.fiscal.position"].search(
+            [("name", "=", self.fp_template.name), ("company_id", "=", self.company.id)]
+        )
```

Comment:
issue: the test is incomplete AFAICS.

The use case we're testing involves a fiscal position template with tax mappings, and a fiscal position with the same tax mappings.

The issue was when those tax mappings were creating duplicate keys, so without tax mappings, the test is incomplete.

Reply:
Ok, I've made one mapping but I'm not sure if its the right way to declare them. I'll leave this review open to check this out.

Reply:
If this assertion passes, then the test is OK:

```suggestion
        fiscal_position = self.env["account.fiscal.position"].search(
            [("name", "=", self.fp_template.name), ("company_id", "=", self.company.id)]
        )
        self.assertEqual(fiscal_position.tax_ids.tax_src_id.name, self.tax_template.name)
        self.assertEqual(fiscal_position.tax_ids.tax_dest_id.name, self.tax_template.name)
```